content/TrueCommand/SystemManagement/TrueNASConfigManage.md

```diff
@@ -48,4 +48,4 @@ You must reset the TrueNAS system to apply the configuration changes.
 
 To delete a backup, click the delete button <i class="material-icons" aria-hidden="true" title="Delete">delete</i> or mark the checkbox and click **Delete Backups**.
```

Comment:
```suggestion icon instead of button?
To delete a backup, click the delete <i class="material-icons" aria-hidden="true" title="Delete">delete</i> icon or mark the checkbox and click **Delete Backups**.
```